da/da_test.go

```diff
@@ -97,26 +92,21 @@ func TestSubmitRetrieve(t *testing.T) {
 	}
 }
 
-func startMockGRPCServ() *grpc.Server {
-	srv := proxy.NewServer(goDATest.NewDummyDA(), grpc.Creds(insecure.NewCredentials()))
-	lis, err := net.Listen("tcp", "127.0.0.1"+":"+strconv.Itoa(7980))
+func startMockDAServ() *proxy.Server {
+	srv := proxy.NewServer("localhost", "7980", goDATest.NewDummyDA())
```

Comment:
any reason for the specific port? we should either define it as a constant and explain why we have a constant port in testing, or we should create unique ports per test.

Reply:
To get ephemeral port we should listen on port `0`. The issue is that we need to connect to this port later, but there is no way to read this port back from proxy server. IMHO we should stick to single port for simplicity, but make host/port constant.

Reply:
Using a const now. Will explain reasoning for static port in comments.